### ISSUE TITLE
MacOS incorrect pip/pip3 version

### ISSUE THREAD

**Issue by OmgImAlexis** [sebhildebrandt/systeminformation#264]
**Describe the bug**
Pip/Pip3 not escaping all data

**To Reproduce**
```bash
➜  core git:(master) ✗ node -r esm
> import si from 'systeminformation'
> si.versions().then(console.log)
Promise { <pending> }
> {
  pip: '',
  pip3: 'from',
}
```

**Current Output**
`''` and `'from'`

**Expected behavior**
`19.0.3`

**Environment (please complete the following information):**
 - systeminformation package version: 4.14.3
 - OS: macOS
 - Hardware Macbook Pro 2010

**Additional context**
```bash
➜  core git:(master) ✗ pip -V
pip 19.0.3 from /usr/local/lib/python2.7/site-packages/pip (python 2.7)
➜  core git:(master) ✗ pip3 -V
pip 19.0.3 from /usr/local/lib/python3.7/site-packages/pip (python 3.7)
```

**Comment by OmgImAlexis** [comment]
https://github.com/sebhildebrandt/systeminformation/blob/cdcc4bc3fd4ee6ab67bea2a84227094a1587dddf/lib/osinfo.js#L585

The offending line.

**Comment by sebhildebrandt** [comment]
@OmgImAlexis ... so yes, this is anyway not correct. Fixed it, version 4.14.4 just published. Thank you!
